R/utils.R

```diff
@@ -76,11 +76,11 @@ get_package_file <- function(pkg = NULL, file_name = NULL) {
   checkmate::assert_string(pkg)
   checkmate::assert_string(file_name)
   base_file <- system.file(file_name, package = pkg)
-  inst_file <- system.file("inst", file_name, package = pkg)
+  inst_file <- file.path(system.file(package = pkg), file_name)
 
   if (file.exists(base_file)) {
     return(base_file)
-  } else if (file.exists(inst_file)) {
+  } else if (file.exists()) {
     return(inst_file)
   } else {
```

Comment:
@gogonzo do we actually need inst_file? Or should I remove lines 79 + 83 + 84 ?

Reply:
agree, remove them as in the compiled package there is no inst directory.